src/uu/shred/src/shred.rs

```diff
@@ -462,6 +466,14 @@ fn wipe_file(
     if force {
         let metadata = fs::metadata(path).map_err_context(String::new)?;
         let mut perms = metadata.permissions();
+        #[cfg(target_family = "unix")]
```

Comment:
Does `#[cfg(unix)]` work too?

Reply:
I'm not sure why `#[cfg(unix)]` works on Windows too.

Reply:
Are you sure? It really should be disabled on windows: https://doc.rust-lang.org/reference/conditional-compilation.html#unix-and-windows